nl/tc_linux.go

```diff
@@ -1022,6 +1029,9 @@ const (
 	__TCA_FLOWER_MAX
 )
 
+const TCA_CLS_FLAGS_SKIP_HW = 1 << 0 /* don't offload action to HW */
```

Comment:
nit:
```
const (
  TCA_CLS_FLAGS_SKIP_HW = 1 << 0 /* dont .... */
  TCA_CLS_FLAGS_SKIP_SW = 1 << 1 /* dont ... */
)
```

also i think this applies to classification (filter) and not the action within a filter. so prehaps change the comment.

Reply:
> so prehaps change the comment.
what to you mean? the comment is copied from iprote2

Reply:
https://github.com/shemminger/iproute2/blob/87dfa589f6c3e9ab1ac2bfb9b5f8591a60320103/include/uapi/linux/pkt_cls.h#L206

Reply:
@e0ne could you align comment with what is defined in iproute2 ? i added link in the comment above.
also while at it, maybe change the const to be a block instead of per line :) not a must by any means.

Reply:
done